deployment_report/model/viya_deployment_report.py

```diff
@@ -600,3 +606,30 @@ def get_cadence_version(resource: KubernetesResource) -> Optional[Text]:
             return cadence_info
         except KeyError:
             return None
+
+    @staticmethod
+    def get_db_info(resource: KubernetesResource) -> Optional[Dict]:
```

Comment:
Could this method be moved to viya_deployment_report_utils.py? It doesn't fit with the other methods in viya_deployment_report.py which are methods to access data in the gathered dictionary. This is formatting data to then be stored in the dictionary, which better aligns with the methods in the utils file.

Reply:
Code updated.